hazelcast-client/src/main/java/com/hazelcast/client/cache/impl/ClientCacheProxy.java

```diff
@@ -472,6 +504,64 @@ public boolean removePartitionLostListener(String id) {
         return getContext().getListenerService().deregisterListener(id);
     }
 
+
+    /**
+     * Subscribe to the event journal for this cache and a specific partition ID.
+     * The method will register the subscription so that the event journal knows which
+     * items have been read by this subscriber. It will return the subscription ID and
```

Comment:
Very likely, comes from the fact that I write down the behaviour all over the place and then forget to update it when it is updated.